src/com/intellij/idea/plugin/hybris/common/HybrisConstants.kt

```diff
@@ -168,6 +168,11 @@ object HybrisConstants {
     const val PLATFORM_LIBRARY_GROUP = "Platform Bootstrap"
     const val PLATFORM_DATABASE_DRIVER_LIBRARY = "Database Drivers"
 
+	const val TOMCAT_WRAPPER_CONFIG_DIR = "conf"
```

Comment:
looks like non-project spacing (tab) formatting settings, can you please reformat?